server/docker/entrypoint.debug.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+cd server
+echo "Waiting for arango to start"
+python wait_for_arango.py
+python manage.py migrate
+pip install debugpy
+cd ..
+touch /tmp/.done.info
+python server/wsgi.py # eventually I want this to start a debugpy server
```

Comment:
The main difference here is that I'm trying to call the wsgi.py server file directly with python instead of going through the uwsgi program.  This will be necessary to use the python debugger, but I haven't yet figured out how to hook up nginx to this 'raw' python server.